eng/scripts/Validate-TypeSpec.ps1

```diff
@@ -11,6 +11,7 @@ param (
 $typespecFolders = @()
 
 $typespecFolders = &"$PSScriptRoot/Get-TypeSpec-Folders.ps1" "$SpecsRepoRootDirectory" "$TargetBranch" "$SourceBranch"
+$typespecFolders = $typespecFolders.Split('',[System.StringSplitOptions]::RemoveEmptyEntries)
```

Comment:
What is the split char there it looks like the empty string.

Reply:
If it is easier you could instead import the other script and call a helper function instead of calling it as a script. That way you could return full lists.

Reply:
this removes the empty/null entries in the array

Reply:
@ckairen: Do you think this is something we can/should fix?  We could open an issue to fix in a separate PR if you want.